components/dash-html-components/dash_html_components_base/__init__.py

```diff
@@ -33,27 +33,27 @@
 
 _js_dist = [
     {
-        "relative_package_path": 'html/{}.min.js'.format(_this_module),
+        "relative_package_path": "html/{}.min.js".format(_this_module),
```

Comment:
ha, we should include this file in our linting / formatting runs. If we did, the new linter would ask for this to become an f-string ;)